src/positioner/src/Positioner.js

```diff
@@ -131,7 +132,7 @@ const Positioner = memo(function Positioner(props) {
         cancelAnimationFrame(latestAnimationFrame.current)
       }
     }
-  }, [previousDimensions.height, previousDimensions.width, update])
+  }, [previousDimensions.height, previousDimensions.width, update, children])
```

Comment:
Nice - glad you tested out adding it to this side effect vs. adding another one in. 💯